storage/acceptance/acls_test.rb

```diff
@@ -0,0 +1,151 @@
+require_relative "helper"
```

Comment:
Please add the copyright header to all the test files. `Copyright 2020 Google LLC` etc.